doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1074,7 +1074,8 @@ the JSON entity representation format (described in a "JSON Entity Representatio
 
 Response:
 
-* Successful operation uses 201 Created. Reponse includes a `Location` header with the URL of the
+* Successful operation uses 201 Created (if upsert option is not used) or 204 No Content (if
+  upsert option is used). Reponse includes a `Location` header with the URL of the
```

Comment:
Only Location header will be present when there was actually an entity creation

Reply:
I think is better to include always the Location heaader. It will be redudant in the case of update, but at the same time it offerst a less variable response interface for clients. It doesn't hurt.